source/wp-content/themes/wporg-parent-2021/patterns/post-navigation.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Title: Post Navigation
+ * Slug: wporg-parent-2021/post-navigation
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"style":{"border":{"top":{"color":"var:preset|color|light-grey-1","width":"1px"}},"spacing":{"padding":{"top":"var:preset|spacing|20","right":"var:preset|spacing|20","bottom":"var:preset|spacing|20","left":"var:preset|spacing|20"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+<div
+	class="wp-block-group"
+	style="
+		border-top-color: var( --wp--preset--color--light-grey-1 );
+		border-top-width: 1px;
+		padding-top: var( --wp--preset--spacing--20 );
+		padding-right: var( --wp--preset--spacing--20 );
+		padding-bottom: var( --wp--preset--spacing--20 );
+		padding-left: var( --wp--preset--spacing--20 );
+	"
+>
+	<!-- wp:post-navigation-link {"type":"previous","showTitle":true,"label":"<?php _e( 'Previous', 'wporg' ); ?>","linkLabel":true} /-->
+	<!-- wp:post-navigation-link {"type":"next","showTitle":true,"label":"<?php _e( 'Next', 'wporg' ); ?>","linkLabel":true} /-->
```

Comment:
Looks like we never set up a linter CI action here, but these are flagged by phpcs (`composer run lint source/wp-content/themes/wporg-parent-2021`).

```suggestion
	<!-- wp:post-navigation-link {"type":"previous","showTitle":true,"label":"<?php esc_attr_e( 'Previous', 'wporg' ); ?>","linkLabel":true} /-->
	<!-- wp:post-navigation-link {"type":"next","showTitle":true,"label":"<?php esc_attr_e( 'Next', 'wporg' ); ?>","linkLabel":true} /-->
```